hazelcast-jet-distribution/src/bin-regular/common.sh

```diff
@@ -28,7 +28,7 @@ if [ "$JAVA_VERSION" -ge "9" ]; then
 fi
 
 if [ "$JET_LICENSE_KEY" ]; then
```

Comment:
I think this actually doesn't need to be in common, since license key is only used on server side, it has no effect for the client. @olukas could you confirm?

Reply:
Yes, that's right. Some time ago it was used also on client side but currently license is configured only for server side.